src/license_manager_agent_ops.py

```diff
@@ -54,6 +54,42 @@ def setup_cache_dir(self):
     def install(self):
         """Install license-manager-agent and setup ops."""
 
+        # Create the license-manager-agent user
+        useradd_cmd = [
+            "adduser",
+            "--system",
+            "--no-create-home",
+            "license-manager",
+        ]
+        subprocess.call(useradd_cmd)
```

Comment:
Should we remove the user when the agent is uninstalled?

Reply:
I think we should!